### PR TITLE
Fix problem with partial app syntax lists

### PR DESCRIPTION
**Pull request by aeslaughter**
(refs #9638)

### CODE REVIEW

File: modules/misc/doc/content/modules/misc/index.md [hunk 1]
```diff
@@ -5,4 +5,4 @@ Documentation for the misc module needs some work...
 
 ## Objects, Actions, and Syntax
 
-!syntax complete group=MiscApp level=3
+!syntax complete groups=MiscApp level=3
```
Comment (sapitts) [inline modules/misc/doc/content/modules/misc/index.md]: Why do some of these apps have level=3 while others use the default level=2?

File: python/chigger/RenderWindow.py [hunk 2]
```diff
@@ -270,6 +270,12 @@ def resetCamera(self):
         for result in self._results:
             result.getVTKRenderer().ResetCamera()
 
+    def resetCameraClippingRange(self):
+        """
+        """
+        for result in self._results:
+            result.getVTKRenderer().ResetCameraClippingRange()
+
```
Comment (sapitts) [inline python/chigger/RenderWindow.py]: Is this intended to be in this merge request as well?

**Review by sapitts — COMMENTED** [review]
Looks pretty good to me, other than the chigger class which I'm not sure belongs in this pull request

File: modules/misc/doc/content/modules/misc/index.md (hunk 1 above)
Comment (aeslaughter) [reply modules/misc/doc/content/modules/misc/index.md]: Whoever created the docs must of wanted it that way. Not sure.

**Comment by sapitts** [comment]
This looks good to me, and hopefully someone like @bwspenc or @permcody can give this merge request the green light

**Review by permcody — APPROVED** [review]

**Pull request merged**